### DISCUSSION TITLE
how to insert images in a submod?

### DISCUSSION THREAD

**Discussion by howltrek** [Monika-After-Story/MonikaModDev#9431] [QA - tech support + general] [answered]
I was working on a full body sprite and realized I don't know how to make it appear in the game.

I humbly ask for help.

**Edited, in case anyone else needs to learn how to do it.**

image _name you want to give the image in your code_ = "path and name with extension of your image"

    show _name you want to give the image in your code_ zorder _number that indicates the depth in which you want your image to appear_

example:

image succubusonthetable = "mod_assets/monika/cg/succubusonthetable.png"

    show succubusonthetable zorder 20

**Comment by Booplicate** [comment, marked as answer]
What kind of sprite? Are you replacing the base?

**Reply by howltrek** [reply]
I want to hide Monika for a moment and replace her with the drawing.

maybe play with the zoom.
![todas las partes grandelocura](https://user-images.githubusercontent.com/90975566/183159384-63ba7de6-45e2-4d12-af0a-ed2751a8c144.png)

**Reply by Booplicate** [reply]
Uh, probably just `hide monika` and then `show your_image`. Replace `your_image` with the name of your image.

**Reply by howltrek** [reply]
thanks

**Reply by howltrek** [reply]
> Uh, probably just `hide monika` and then `show your_image`. Replace `your_image` with the name of your image.

parameters are missing for the "show" function, it doesn't give me an error, but it doesn't do anything either:/

**Reply by Booplicate** [reply]
Show your code?

**Reply by howltrek** [reply]
image succubusonthetable = "mod_assets/monika/cg/succubusonthetable.png"

        show succubusonthetable zorder 20

Thanks.
I just needed to first establish a name for the image and its path.

now it works perfect.
